src/keyboard.cpp

```diff
@@ -1398,3 +1401,25 @@ void InputMapper::CPCkeyFromJoystickAxis(SDL_JoyAxisEvent jaxis, dword *cpc_key,
 }
 
 InputMapper::InputMapper(t_CPC *CPC): CPC(CPC) { }
+
+void applyKeypress(dword cpc_key, byte keyboard_matrix[], bool pressed) {
+    if ((!CPC.paused) && (static_cast<byte>(cpc_key) != 0xff)) {
+		if (pressed) {
```

Comment:
Yes, I was thinking of using clang-format to homogenize the style on the whole project. I'll have a look at it and ask you a good time to do it so that it doesn't conflict with some code you do :-)